src/verso/Verso/Doc/Concrete.lean

```diff
@@ -136,7 +137,7 @@ be used to thread state between the separate top level blocks. These environment
 the state that needs to exist across top-level-block parsing events.
 -/
 structure DocElabEnvironment where
-  ctx : DocElabContext := ⟨.missing, mkConst ``Unit⟩
+  ctx : DocElabContext := ⟨.missing, mkConst ``Unit, false⟩
```

Comment:
Perhaps these use sites should now use named field syntax, so the purpose of the Boolean is more apparent? Or, alternatively, could the Boolean be replaced by a special-purpose enumeration like:
```
inductive AllowUndefinedRefs where
  | allowUndefinedRefs
  | forbidUndefiendRefs
```

For functions, we can hover them and see parameter names, but that doesn't work for anonymous constructor syntax.

Reply:
Went with this:

```
/--
Specifies the elaboration behavior of inline references in Verso.
-/
inductive RefsAllowed : Type where
  /--
  A footnote ref like `[^note]` or a link ref like {lit}`[wikipedia]` are treated as an error if the
  current {lit}`PartElabM` state does not contain a definition for the ref.
  -/
  | onlyIfDefined
  /--
  Undefined link and footnote references in inline text are permitted with no warning.
  -/
  | always
```

can/will revise if you want to nitpick post-merge